src/nupic/database/ClientJobsDAO.py

```diff
@@ -55,8 +55,7 @@ def _getLogger():
   needs to be initialized by the host app, and that usually happens after
   imports
   """
-  return logging.getLogger(
-    ".".join(['com.numenta', _MODULE_NAME, ClientJobsDAO.__name__]))
+  return logging.getLogger(__name__)
```

Comment:
@scottpurdy The reason I held off doing this is that the logExceptions decorator also expects a callback, requiring us to redefine and change all references to the decorator as well. Though, looking into it, it appears that other than the test for that decorator, this file is the only time it is used. Should I go ahead and change it too? I suppose the other approach would be to use a lambda when calling it in this file.